lib/modules/manager/vendir/extract.ts

```diff
@@ -63,6 +62,20 @@ export function extractGitSource(
   };
 }
 
+export function extractGithubReleaseSource(
+  githubRelease: GithubReleaseDefinition,
+): PackageDependency | null {
+  const registryUrl = 'https://github.com/' + githubRelease.slug;
+  return {
+    depName: githubRelease.slug,
+    packageName: githubRelease.slug,
+    depType: 'GithubRelease',
+    currentValue: githubRelease.tag,
+    registryUrls: [registryUrl],
```

Comment:
```suggestion
```
we only need package name. registry url defaults to GitHub. also it should not contain the repo path.

Reply:
Updated to remove